### DISCUSSION TITLE
Custom Icons and Tooltip - issue in documentation ?

### DISCUSSION THREAD

**Discussion by danielczestki** [pimcore/pimcore#8836] [Q&A / Support]
I want to create custom icons and tooltip with this tutorial:
https://pimcore.com/docs/pimcore/6.8/Development_Documentation/Tools_and_Features/Custom_Icons.html

What I see in code examples is that there is a use of **AppBundle\Website\Tool\ForceInheritance** but it does not exist in my project.

Are these examples correct ? What should I do with it ?

**Comment by NiklasBr** [comment]
I had the same question, but after a lot of digging it appears to be an undocumented [custom class from the Demo repository](https://github.com/pimcore/demo/blob/10.x/src/Website/Tool/ForceInheritance.php).
